debian/changelog

```diff
@@ -1,4 +1,7 @@
 python-oq-hazardlib (0.20.0-0~precise01) precise; urgency=low
+  [Yen-Shin Chen]
+  * Fixed the rupture count in simple fault while multiple hypocenter locations
+  or slip directions assigned.
 
```

Comment:
Write *in a single line*

`* Fixed the rupture count in simple fault sources with multiple hypocenter locations and/or slip directions`